news/2018-05-21-project-loved-week-of-may-20th.md

```diff
@@ -0,0 +1,242 @@
+
+If you think, the first half is hard enough, well.... The second half is almost the same, but having a noticeable density increase that will make everyone suffer, especially stamina wise. Definitely a map worth enough for Loved.
+
+[![](/wiki/shared/news/2018-05-21-project-loved-week-of-may-20th/mania/seven-colors.jpg)](https://osu.ppy.sh/community/forums/topics/749084)
+
+[UNDEAD CORPORATION - Seven Colors](https://osu.ppy.sh/beatmapsets/288138) by [Nwolf](https://osu.ppy.sh/users/1910766) et al.  
+*written by [aitor98](https://osu.ppy.sh/users/3154852)*
+
+Being a multimode set, [Seven Colors by UNDEAD CORPORATION](https://osu.ppy.sh/beatmapsets/288138) brings us a really good 7K difficulty set made by [LordRaika](https://osu.ppy.sh/users/3265023) and [Arzenvald](https://osu.ppy.sh/users/3027421). It mostly features a good mix between low-mid density chordstreams and Long Note sections, followed by sightreadable Scroll Velocities throughout the entire map at 200 BPM for almost 3 minutes and a half, which means two things, stamina and consistency.
+
+This map was also featured in the 7K osu!mania World Cup 2018 Quarterfinals mappool and it is good enough to give it a shoot at least. Good luck!
+
+[![](/wiki/shared/news/2018-05-21-project-loved-week-of-may-20th/mania/outside.jpg)](https://osu.ppy.sh/community/forums/topics/749085)
+
+[Bill wurtz - outside](https://osu.ppy.sh/beatmapsets/735580) by [Nivrad00](https://osu.ppy.sh/users/1984634)  
+*written by [Kamikaze](https://osu.ppy.sh/users/2124783)*
+
+Our last chart in the list, comes from the charter that at this point, everyone knows. The most unique persona in osu!mania's charting scene - [Nivrad00](https://osu.ppy.sh/users/1984634). This time we are featuring one of his works made specifically for this year's 7K Mania World Cup - [Outside](https://osu.ppy.sh/beatmapsets/735580).
+
+This jazzy tune by even more unique artist - Bill Wurtz is complimented incredibly well by the tasteful and silly usage of SV gimmick, including some funky white line gimmicks and fairly artistic patterning, combining abrasive LN patterning with wide variety of rice including weird bursts and tough to read minijacks. It's really going Outside the boundaries of conventional charting and as such deserves a spot in the Loved category!
+
+---
+
+From everyone on the Project Loved team, we hope you enjoy this week's selection, and look forward to the next! The winning beatmaps will hit the [beatmap listing](https://osu.ppy.sh/beatmapsets?s=8) sometime next week and will be announced via the [osugame twitter](https://twitter.com/osugame).
```

Comment:
Can you apply the edits(reviews from this line) from the old PRs? This seems to be reverted back every time.